.github/workflows/pythonbuild.yml

```diff
@@ -78,33 +78,39 @@ jobs:
         with:
           path: ./wheelhouse/*.whl
 
-  build_wheels_manylinux1:
+  build_wheels_manylinux:
     needs: [test_python]
-    name: Build wheel with manylinux1
+    name: Build wheels on ${{matrix.arch}} for ${{matrix.python_tag}}
     runs-on: ubuntu-18.04
     strategy:
       fail-fast: false
-    env:
-      CIBW_BUILD: cp*
-      CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
-      CIBW_MANYLINUX_I686_IMAGE: manylinux1
+      matrix:
+        arch: [auto, aarch64, ppc64le, s390x]
+        python_tag: [ "cp35-*", "cp36-*", "cp37-*", "cp38-*", "cp39-*"]
```

Comment:
1) I think this rebuilds some of the wheels that are already built in the build_wheels job for linux.
2) I am not sure how cibuildwheel decides which manylinux it uses and which we actually need. Right now I build many things for manylinux1 and 2010 and upload them both, but I have absolutely no clue whether both are required and when which of the two is used. Are you more familiar with this?
3) I really like having single builds for each configuration, since we can have 20 builds in parallel on github actions. Could you update the build_wheels job to do this as well? e.g. the Ubuntu config currently takes over 20 min, so splitting it in a similar way would probably speed this up

Reply:
Please find my response inline:
> 1. I think this rebuilds some of the wheels that are already built in the build_wheels job for linux.

Yes, it was building the linux x86_64 wheels which I have removed from buid_wheels. So now all the linux builds are running under build_wheels_manylinux.
> 2. I am not sure how cibuildwheel decides which manylinux it uses and which we actually need. Right now I build many things for manylinux1 and 2010 and upload them both, but I have absolutely no clue whether both are required and when which of the two is used. Are you more familiar with this?

Cibuildwheel takes care of adding manylinux tag i.e. manylinux1/manylinux2010 as per eligibility.
> 3. I really like having single builds for each configuration, since we can have 20 builds in parallel on github actions. Could you update the build_wheels job to do this as well? e.g. the Ubuntu config currently takes over 20 min, so splitting it in a similar way would probably speed this up

Done.